lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/new-p1-identity.yaml

```diff
@@ -0,0 +1,1275 @@
+name: New P1 Identity
```

Comment:
Aside from the first few lines this is just a copy/paste of the existing P2 journey map

Reply:
I was wondering if we could just reuse the p2 journey map at least for now (possibly renaming it `new-identity.yaml`) - the P1 journey is practically identical other than the no photo id route so I'm a bit nervous about having a duplicate to maintain. Or maybe we need to think about a way of packaging up the routes so we can reuse them

Reply:
Sorry, I misunderstood what you meant by "re-use" :)